src/pages/books/Books.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import BookCard from '../../components/BookCard';
+import Style from './Books.module.css';
+import BookForm from '../../Forms/book-form/BookForm';
+
+/*  eslint-disable react/jsx-curly-brace-presence */
+export default function Books() {
+  return (
+    <>
+      <div className={`${Style.PageContainer}`}>
+        <BookCard />
+        <BookCard />
+      </div>
+      <div className={`${Style.Spacer}`} />
+      <BookForm />
+    </>
```

Comment:
[OPTIONAL] This may be marked as optional but it must be implement it has only been marked as optional because the previous code reviewer didnt spot this issue. It is against microverse policy to disable linters in this case you disabled eslint in this file. It would be nice if you enabled eslint.